projects/aca-playwright-shared/src/page-objects/components/search/search-input.component.ts

```diff
@@ -43,9 +43,19 @@ export class SearchInputComponent extends BaseComponent {
   }
 
   async performDoubleClickFolderOrFileToOpen(name: string): Promise<void> {
+    await this.waitForSearchForResult(name);
     await this.getCellLinkByName(name).waitFor({ state:'visible', timeout: timeouts.normal });
     await this.getCellLinkByName(name).dblclick();
     await this.spinnerWaitForReload();
   }
 
+  async waitForSearchForResult(name: string): Promise<void> {
+    const retry = 0;
```

Comment:
This code looks a bit wrong, you're using `retry` as a while condition but it's never updated inside so it basically mean that the loop executes one time, is this correct?

Reply:
corrected in commit 903a093f31b44e0074d08b21bdf484e92c40c9ed